src/main/java/com/brightcove/castlabs/client/CastlabsClient.java

```diff
@@ -24,14 +24,15 @@
 
 import com.brightcove.castlabs.client.request.IngestKeysRequest;
 import com.brightcove.castlabs.client.response.IngestAssetsResponse;
+import com.brightcove.castlabs.client.request.AddSubMerchantAccountRequest;
+import com.brightcove.castlabs.client.response.AddSubMerchantAccountResponse;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 /**
  * Client for interacting with the Castlabs key ingestion API.
- * 
- * @author Scott Kidder
  *
+ * @author Scott Kidder
```

Comment:
The `@author` Javadoc tag has fallen out of favor, although some IDE templates - in this case, Eclipse - still populate the `@author` tag in new files. No hard feelings, remove that cruft.